README.md

```diff
@@ -1,5 +1,5 @@
 [![Gem Version](https://badge.fury.io/rb/raygun.svg)](http://badge.fury.io/rb/raygun)
-<img src="https://raw.github.com/carbonfive/raygun/main/marvin.jpg" align="right"/>
+<img src="https://user-images.githubusercontent.com/896684/168395357-1fe805fd-2fd0-4f94-b263-1dc81f20bc6c.png" align="right"/>
```

Comment:
❔ What do you think about updating the image in the repo itself? I kind of like the idea of the image traveling with the source code so that you don't need to be online and relying on GitHub to serve the image. (E.g. when viewing the README locally in one's editor)

Reply:
Oops, totally missed that. Thanks Matt! One sec…

Reply:
👍 fixed (and I touched up the new png so it still includes the artist's signature)

Reply:
Wait, I didn't fix the URL 🤦

Reply:
OK, now I updated the image src to be a relative path, so it's working locally in my markdown preview. 

<img width="1792" alt="local demo" src="https://user-images.githubusercontent.com/896684/168398962-a53cb547-513c-4d49-bb2d-3fad1c499a9c.png">

And Github is picking it up in the "rich diff", 

<img width="1395" alt="image" src="https://user-images.githubusercontent.com/896684/168399085-87ddf9fe-47f2-4dc5-bc73-808625e63cce.png">